src/analyzer/vulnerabilities/arbitrary_from_in_transferfrom.rs

```diff
@@ -0,0 +1,124 @@
+use std::collections::HashSet;
+
+use solang_parser::pt::{self, Expression, Loc};
+use solang_parser::{self, pt::SourceUnit};
+
+use crate::analyzer::ast::{self, Target};
+
+pub fn _arbitrary_from_in_transferfrom_vulnerability(source_unit: SourceUnit) -> HashSet<Loc> {
+    //Create a new hashset that stores the location of each vulnerability target identified
+    let mut vulnerability_locations: HashSet<Loc> = HashSet::new();
+
+    //Extract the target nodes from the source_unit
+    let target_nodes =
+        ast::extract_target_from_node(Target::FunctionDefinition, source_unit.into());
+
+    //For each target node that was extracted, check for the vulnerability patterns
+    for _node in target_nodes {
```

Comment:
Can you remove the `_` in front of the `_node` since `node` is being used?

Reply:
Done